feature_campaigns/src/main/java/cm/aptoide/pt/feature_campaigns/MMPLinkerCampaign.kt

```diff
@@ -0,0 +1,30 @@
+package cm.aptoide.pt.feature_campaigns
+
+class MMPLinkerCampaign(
```

Comment:
I don't see it's used anywhere it this PR.
Let's add it in the PR for MMP linker, not in this one.